Gems/MiniAudio/Code/Source/Clients/MiniAudioPlaybackComponentController.cpp

```diff
@@ -232,6 +338,23 @@ namespace MiniAudio
                     ma_sound_set_min_distance(m_sound.get(), m_config.m_minimumDistance);
                     ma_sound_set_max_distance(m_sound.get(), m_config.m_maximumDistance);
                     ma_sound_set_attenuation_model(m_sound.get(), static_cast<ma_attenuation_model>(m_config.m_attenuationModel));
+                    ma_sound_set_directional_attenuation_factor(m_sound.get(), m_config.m_directionalAttenuationFactor);
+
+                    // Set the forward direction for the sound source
+                    if (!m_config.m_fixedDirection)
+                    {
+                        AZ::Transform worldTm = AZ::Transform::CreateIdentity();
+                        AZ::TransformBus::EventResult(worldTm, m_entityComponentIdPair.GetEntityId(), &AZ::TransformBus::Events::GetWorldTM);
+                        ma_sound_set_direction(m_sound.get(), worldTm.TransformVector(AZ::Vector3::CreateAxisY(1.f)).GetX(),
```

Comment:
Anywhere you use the transformVector() with a unit vector you can probably replace with the GetBasisXYZ equivalent to avoid the math.

Reply:
See 87d1400dfa20a3850ea7f1a1a4712de7730b4318